lib/src/visitor/async_evaluate.dart

```diff
@@ -1733,37 +1785,51 @@ final class _EvaluateVisitor
     }
   }
 
-  Future<Value?> visitIncludeRule(IncludeRule node) async {
-    var nodeWithSpan = AstNode.fake(() => node.spanWithoutContent);
-    var mixin = _addExceptionSpan(node,
-        () => _environment.getMixin(node.name, namespace: node.namespace));
+  /// Evaluate a given [mixin] with [arguments] and [contentCallable]
+  Future<void> _applyMixin(
+      AsyncCallable? mixin,
+      UserDefinedCallable<AsyncEnvironment>? contentCallable,
+      ArgumentInvocation arguments,
+      FileSpan span,
+      FileSpan spanWithoutContent) async {
```

Comment:
Should those spans really be provided as is or should a node be provided to keep them lazy in case there is no error ?

Reply:
`IncludeRule` isn't possible afaict because `meta.apply` doesn't have access to the rule including it. I made this change in https://github.com/sass/dart-sass/pull/2073/commits/f1c1d5af1cc1408b80fc240c3ba42077029d2e88 where I just use a regular `AstNode`.